doc/develop/extra_facets.rst

```diff
@@ -0,0 +1,111 @@
+.. _extra_facets:
```

Comment:
How would you feel about adding this section to the chapter we have on configuration instead? https://esmvaltool--1124.org.readthedocs.build/projects/ESMValCore/en/1124/quickstart/configure.html